exercises/practice/spiral-matrix/spiral_matrix.rb

```diff
@@ -0,0 +1,17 @@
+=begin
+Write your code for the 'Spiral Matrix' exercise in this file. Make the tests in
+`spiraL_matrix_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/spiral-matrix` directory.
+=end
+
+class SpiralMatrix
+  def initialize
+    # Your code here
+  end
```

Comment:
I have a preference not to have methods defined in stubs with the wrong signatures (e.g. here initialize needs to take a size). And equally, I'd rather not force the implementation by having a param defined, so I think my preference is just to remove this method and the the one below. 

But please tell me if there's a specific reason you felt they would be good to include.

Reply:
This was how I'd been asked to implement it in the Dnd Character exercise MR, was copying here, but happy to change whatever!

Reply:
Yeah, I think you should remove the code and just leave the comment at the top please. That's how all the other practice exercises are implemented (e.g. https://github.com/exercism/ruby/blob/main/exercises/practice/binary-search/binary_search.rb :)

Reply:
I think it was a focus on providing a comment so that we did not submit an empty file.  Leave the practice exercises as mostly unguided.